src/current/_includes/releases/v23.2/v23.2.18.md

```diff
@@ -0,0 +1,14 @@
+## v23.2.18
+
+Release Date: December 26, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-9-sql-language-changes">SQL language changes</h3>
+
+- Added the `legacy_varchar_typing` session setting. When set to `on`, type-checking and overload resolution for comparisons involving `VARCHAR` columns ignore the newly added `unpreferred` overload preference. The `unpreferred` preference introduced a change in `VARCHAR` typing behavior. For more information, see [#133037][#133037] and [#137837][#137837]. The `legacy_varchar_typing` session setting is `on` by default. [#137943][#137943]
+
```

Comment:
s/137943/137945/:

- v24.3.2 backport is https://github.com/cockroachdb/cockroach/pull/137943
- v23.2.18 backport is https://github.com/cockroachdb/cockroach/pull/137945